www/source/blog/2018-05-11-changes-in-the-0.56.0-supervisor.html.md

```diff
@@ -0,0 +1,64 @@
+
+With the 0.56.0 release, we have transitioned away from file-based communication to a defined TCP protocol between `hab` and a Supervisor process. Defining a protocol brings added rigor to the Supervisor codebase, which brings additional safety, enhanced backward-compatibility between releases going forward, and lays the groundwork for features that would otherwise have been impossible to implement. While this change has been introduced in a way that it can be largely transparent to current Habitat users in the near term, there are some manual steps that current users will need to take to ensure a smooth transition. We will highlight these changes now.
+
+## Install `core/hab/0.56.0` on _all_ Supervisor Machines
+
+Since communication with the new Supervisor takes place strictly via TCP, it needs a client that also speaks this same protocol. That client is the `hab` CLI tool. This means that in order to continue interacting with a Supervisor, you will need to manually install `core/hab/0.56.0` on all machines running the 0.56.0 Supervisor; if your Supervisors are configured to auto-update, then they will automatically begin running the 0.56.0 release, but you will still need to upgrade `core/hab`. In a forthcoming release, we will begin bundling these into a single package, but for now, you must take this manual step.
+
+## Revised Command Suite
+
+With this reworking of the interaction model of Habitat, we had the opportunity to streamline and clarify the suite of commands for interacting with services. One of the most frequently-asked questions about Habitat has been "what's the difference between `hab start` and `hab load`?". This has been difficult to clearly explain in the past, but now we have very clearly differentiated _all_ the commands, in a way that we hope will be more easy to reason about.
+
+Please review any scripts or automation you have that make use of these commands and update them as necessary.
+
+`hab svc load` is how you inform a Supervisor to manage a new service. This will also start the service once it has been loaded.
+
+`hab svc unload` is how you remove a service from the Supervisor. If the service is currently running, it will be stopped before being unloaded.
+
+`hab svc start` is how you start a previously-stopped, loaded service.
+
+`hab svc stop` is how you stop a currently-running (and thus, loaded) service.
+
+`hab sup run` is now the only way to start a Supervisor. As a convenience, `hab sup run` can also accept arguments that load and start a single service. This means that you are still able to start a Supervisor and service in a single command.
```

Comment:
Are there/will there be updated CLI docs for `hab sup run` we can link to from here?